src/server/pkg/storage/chunk/util.go

```diff
@@ -19,7 +21,11 @@ func NewTestStorage(t testing.TB, db *sqlx.DB, tr track.Tracker, opts ...Storage
 
 // NewTestStore creates a store for testing.
 func NewTestStore(t testing.TB, db *sqlx.DB) MetadataStore {
-	db.MustExec(schema)
+	ctx := context.Background()
+	tx := db.MustBegin()
+	tx.MustExec(`CREATE SCHEMA IF NOT EXISTS STORAGE`)
```

Comment:
I want to avoid the conditional creates in the production setup SQL, so this has to be added to the test methods